src/cmd/index.js

```diff
@@ -67,11 +67,17 @@ module.exports = async function run (pkg) {
     })
 
   program
-    .command('create [propertyId]')
+    .command('create [propertyId] [name] [split] [template]')
     .usage(chalk.gray(`
     Create using propertyId:
     qubit create 1010
 
+    Crete using propertyId and new experience name:
```

Comment:
Crete

Reply:
Prefer another island like Santorini?

Fixed the silly mistake :)